micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/RoleLanding/RoleLandingPage.js

```
@@ -0,0 +1,70 @@
+import { Card, CardText, CardHeader, Button , Header} from "@egovernments/digit-ui-components";
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import { externalConfig } from "./RoleLandingConfig";
+import YoutubeVideo from "../Landing/YoutubeVideo";
+
+const RoleLandingComponent = () => {
+    const { t } = useTranslation();
+    const history = useHistory();
+
+    const handleButtonClick = (action) => {
+        const url= '/'+window.contextPath+action
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)